tests/unit/translations-test.js

```diff
@@ -1,32 +0,0 @@
-import { module, test } from 'qunit';
```

Comment:
This bit scares me. I don't see a solution though other than an open issue on the repo so I guess I'm just going to need to be more careful!

Reply:
alternatively, we could look into yaml specific diff tools and run them on travis out-of-band.